plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/OAuthDetailView.java

```diff
@@ -212,6 +212,11 @@ public String toString() {
 		}
 
 		@Override
+		public void widgetDefaultSelected(SelectionEvent e) {
+			widgetDefaultSelected(e);
+		}
```

Comment:
Actually, Rob noticed it but I tend to be more verbose about actually putting comments in JIRAs and PRs. :D